Samples.sln

```diff
@@ -0,0 +1,189 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00

```

Comment:
I am relatively certain that the YAML build files are only building OpenConsole.sln and not any .sln file. Do you want to add this to be built in CI perhaps in parallel or on a separate phase just to make sure it keeps working? (I would say put it on the rolling build... but we're about to nuke that and move its checks to CI since we never see the rolling build failures...)

Reply:
Nah, I'm pretty explicitly not putting it in CI. It's literally just for local scratch testing.